### DISCUSSION TITLE
Posts do not show up in a blog/feed view for other users.

### DISCUSSION THREAD

**Discussion by TheSenPie** [writefreely/writefreely#794] [Questions / Support] [answered]
### Describe the bug
When publishing a post in a public blog let's say `tech`, those posts under `tech` blog are not visible to other users. That is when author opens `https://example.com/tech`, they can see their posts in list, however when other users visit the same link to view all articles available under the blog, they do not show up. Same when link is opened in incognito mode. Refreshing doesn't help. Also, posts are not federated.

### Steps to reproduce (if necessary)
1. Write a post, publish it.
2. Go to that blog page in incognito.
3. Posts are not showing up.
4. Check mastadon profile corresponding to that blog, again posts are not showing up.

### Expected behavior
1. Should list all posts under given blog.
2. Should be visible from mastadon profile.

### Application configuration
 - Multi-user mode
 - Database: mysql
 - Open registration: yes
 - Federation enabled: yes

**Version or last commit**:
I am using FreeBSD, so installed the latest available package with version 0.14.0.

### Update
After whole day passed, one of the posts become visible in the website. However, none of the posts are yet federated.

**Comment by thebaer** [comment]
Hi, do you have an exact URL I can look at to try and replicate this? I'm not quite sure I understand what's going on.

**Comment by TheSenPie** [comment]
Hej @thebaer, this is the website https://blog.sandm.cc/. I will open the registration temporarily for you. You can make account and try making posts.

**Comment by thebaer** [comment, marked as answer]
Thanks, I'm able to replicate the issue there. Based on this, I'm guessing that your timezone between your machine and database aren't synced up. The posts probably aren't showing up in incognito because they seem to have been published in the future, so they won't come up for anyone who isn't the owner.

Have you set both your machine and database timezone to UTC, as mentioned in the install docs?

**Reply by TheSenPie** [reply]
You are right! I completely missed that part and set `/etc/localtime` to point to `/usr/share/zoneinfo/Asia/Yerevan`. Thank you so much!

For anyone wondering how I fixed it, I run following command.
`
ln -sf /usr/share/zoneinfo/UCT /etc/localtime
`
